StrongInject.Tests.Unit/OwnedGeneratorTests.cs

```diff
@@ -0,0 +1,95 @@
+﻿using FluentAssertions;
```

Comment:
I don't mind splitting up the generator tests into separate files, but if so could you fix GeneratorTestsUpdater so that it fixes this file too? That way if I make a change which affects all generated code, I don't need to update every single test individually.

Thanks!

Reply:
Before I do this, is this grouping okay with you? I like it because it keeps the files together and communicates "this is a subset."

![image](https://user-images.githubusercontent.com/8040367/128700110-5010acb9-a98c-4474-8772-3616403606d3.png)

Reply:
Yep, perfect

Reply:
It does make it harder to say "run all tests relating to `Owned<T>` injection." Maybe a folder would be a good idea?

Reply:
Also sounds good

Reply:
Now I'm realizing that the only things in the project are DependencyCheckerVisitorTests and GeneratorTests which both inherit from TestBase, or utilities used by these classes. A folder makes less sense to me now.

Reply:
Ah, maybe the folder could be called `Output` and anything in the folder can be updated by the updater?

Reply:
I don't really like the name Output - seems confusing to me.

Reply:
Yeah. Since there didn't seem to be a need to filter which test classes the updater should look at, I didn't introduce any form of grouping. The latest push introduces the ability (worked on my machine™) to update generator output in a different test class and file name.

Reply:
Yeah. Since there didn't seem to be a need to filter which test classes the updater should look at, I didn't introduce any form of grouping. The latest push introduces the ability (worked on my machine™) to update generator output in a different test class and file name.